app/routers/request_models.py

```diff
@@ -51,6 +51,7 @@ class CreateTokenRequest(BaseModel):
     disallowed_regions: Optional[List[str]] = []
     is_smart_contract_token: bool = False
     token_update: bool = False
+    token_add_update : bool = False
```

Comment:
@Prannav-k what will be the behaviour when `token_update` and `token_add_update` are present? also with delete. ideally only 1 flag of type string or int should be used which can be something like
- 1 for add
- 2 for modify
- 3 for delete